packages/spatial/src/renderer/constants/ObjectLayers.ts

```diff
@@ -56,5 +56,7 @@ export const ObjectLayers = {
   // transform gizmo
   HighlightEffect: 11 as const,
 
+  UVOL: 30 as const,
+
   AssetPreview: 31 as const
 } as Record<string, number>
```

Comment:
We can't go over 31 atm due to maxBitWidth. We should consider refactoring some of these if we can